literature/briega-martos_2021_cation_48/briega-martos_2021_cation_48_p2_f1_Li_2.3.yaml

```diff
@@ -10,62 +8,62 @@ electrochemical system:
       material: Pt
       shape: coiled wire
     reference electrode:
-      source:
-        supplier: none
       type: RHE
     working electrode:
-      crystallographic orientation: 111
+      crystallographic orientation: '111'
       geometric electrolyte contact area:
-        unit: cm-2
-        value: null
+        unit: 1 / cm2
       material: Pt
-      preparation procedure: null
       shape:
         diameter:
           unit: mm
           value: 2
         height:
           unit: mm
-          value: null
         type: bead
-      source:
-        LOT: null
-        supplier: null
   electrolyte:
     components:
     - concentration:
         unit: mol / l
         value: 0.04
       name: LiF
       source:
-        supplier: Merck, Suprapur, 99.99%
-      sum formula: LiF
+        supplier: Merck, Suprapur
```

Comment:
Isn't the supplier just Merck? and Suprapur the brand name?

Anyway, I am also not sure what exactly is the meaning "supplied purity" and why this is not only called purity. And then why is the subkey "grade" needed at all. Couldn't it just be:

```
purity:
  value: 99.99
  unit: pct
```

But it also doesn't really matter. Just something I stumbled upon.

Reply:
Merck for example calls "Suprapur©" a grade.
Maybe our purity can hold a grade string.